### PR TITLE
Fix spelling error ("sript" vs "script")

### PR DESCRIPTION
**Pull request by totten**
Overview
----------------------------------------
Fix spelling error in code comment

Before
----------------------------------------
Code comment spelled 'script' incorrectly

After
----------------------------------------
Code comment spells 'script' correctly

Technical Details
----------------------------------------
Code comment only

Comments
----------------------------------------
It has a 'c'

### CODE REVIEW

**Comment by colemanw** [comment]
After careful review, I agree that "script" has a C.
That's good enough for me!
C is for sCript

File: CRM/Utils/System/Drupal.php [hunk 1]
```diff
@@ -265,6 +265,7 @@ public function addScript($code, $region) {
     return TRUE;
   }
 
+
```
Comment (colemanw) [inline CRM/Utils/System/Drupal.php]: Oh no! Jenkins not like whitespace! Whitespace not start with C.

Comment (wmortada) [reply CRM/Utils/System/Drupal.php]: Cookie monster eat whitespace. Yum!!

Comment (totten) [reply CRM/Utils/System/Drupal.php]: Please ensure you get consent before eating a cookie.

https://www.cookielaw.org/the-cookie-law/

**Comment by eileenmcnaughton** [comment]
@totten even though an extra test failed here I'm still going with 'unrelated'

**Pull request merged**